jenkins/ring-install.sh

```diff
@@ -71,15 +90,22 @@ function add_source {
 }
 
 function add_source_centos {
+    local centos_version=${os_RELEASE:0:1}
     set +x
     sudo sh -c "cat <<-EOF >/etc/yum.repos.d/scality${RING_VERSION}.repo
 [scality-base]
 name=Centos6 - Scality Base
```

Comment:
Change the name of the repo. I suggest "Centos - Scality Base", no version number, no problemo :)